nodeoperators/nodelist.json

```diff
@@ -2937,12 +2937,36 @@
     },
     {
         "Role": "access",
-        "Address": "flow-mainnet-1.blockchain.circle.com:3569",
+        "Address": "flow-mainnet-0.prod.blockchain.circle.com:3569",
```

Comment:
One network address change and 3 new AN for Circle. All 4 are not staked with the correct addresses